src/chess_piece_move_validation_tests/queen/queen.ino

```diff
@@ -0,0 +1,85 @@
+/**
+ * @file queen.ino
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Test for ECPQueen isMoveValid method.
+ * @version 0.1
+ * @date 2024-12-02
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+#include <Wire.h>
+
+#define BAUD_RATE 9600
+
+Dezibot dezibot = Dezibot();
+
+void setup() {
+    Serial.begin(BAUD_RATE);
+    dezibot.begin();
+    delay(500);
+}
+
+void loop() {
+    Serial.println("\n=== STARTING TEST ===\n");
+
+    Serial.println("Testing queen...");
+    testQueen();
+
+    Serial.println("\n=== TEST DONE ===");
+    Serial.println("Sleeping for 10 seconds...\n");
+
+    delay(10000);
+}
+
+void testQueen() {
+    const ECPChessField initialField = { D, 4 };
+    ECPQueen queen = ECPQueen(dezibot, initialField, true);
+
+    // valid moves
+    Serial.println("Testing valid moves...");
+    ECPChessField validFields[] = {
+                                      { D, 8 },                               { H, 8 },
+        { A, 7 },                     { D, 7 },                     { G, 7 },
+                  { B, 6 },           { D, 6 },           { F, 6 },
+                            { C, 5 }, { D, 5 }, { E, 5 },
+        { A, 4 }, { B, 4 }, { C, 4 }, /* fig */ { E, 4 }, { F, 4 }, { G, 4 }, { H, 4 },
+                            { C, 3 }, { D, 3 }, { E, 3 },
+                  { B, 2 },           { D, 2 },           { F, 2 },          
+        { A, 1 },                     { D, 1 },                     { G, 1 }
+    };
```

Comment:
Please move `};` to the line above for consistency.

Reply:
Fixed in 6ae4f97418193f6045f94624918fad9938d50a31